src/installer/managed/Microsoft.NET.HostModel/AppHost/AppHostSigningException.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.NET.HostModel.AppHost
+{
+    /// <summary>
+    /// Unable to sign the apphost binary.
+    /// </summary>
+    public class AppHostSigningException : AppHostUpdateException
```

Comment:
`sealed`?

Reply:
Also consider putting all these tiny types into one file.

You could also introduce an `internal` constructor to prevent arbitrary new exceptions from being defined.